components/hab/static/template_gitignore

```diff
@@ -0,0 +1,2 @@
+.DS_Store
+results/
```

Comment:
yeah I think its a really good question and think @cnunciato 's idea to let the user manage through a top level .gitignore  is likely the safest approach. Searching for ancestor ignore files feels a bit too "clever" for now.